src/open_samus_returns_rando/specific_patches/chozo_seal_patches.py

```diff
@@ -165,8 +166,9 @@ def update_item_seals(editor:PatcherEditor):
             scenario.raw.actors[16][seal]["type"] = "chozoseal"
 
         # Dependencies
-        for asset in editor.get_asset_names_in_folder("actors/props/chozoseal"):
-            editor.ensure_present_in_scenario(scenario_name, asset)
+        for level_pkg in editor.get_all_level_pkgs():
```

Comment:
This is also iterating over all level pkgs for each chozo seal which is definitely unnecessary.